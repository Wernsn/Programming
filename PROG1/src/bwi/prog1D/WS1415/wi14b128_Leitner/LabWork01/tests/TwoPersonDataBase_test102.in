2
Bugs
Bunny
19570401
3
0
