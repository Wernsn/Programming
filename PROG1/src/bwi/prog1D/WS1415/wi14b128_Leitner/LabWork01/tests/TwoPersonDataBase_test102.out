Welcome to TwoPersonDataBase!

available options:
1) enter data of person1
2) enter data of person2
3) list by firstname
4) list by lastname
5) list by age
0) quit
your choice: enter firstname (2)enter lastname (2)please enter date of birth (2) (yyyymmdd)
available options:
1) enter data of person1
2) enter data of person2
3) list by firstname
4) list by lastname
5) list by age
0) quit
your choice: 
2: BUGS       BUNNY         Monday, 04/01/1957
available options:
1) enter data of person1
2) enter data of person2
3) list by firstname
4) list by lastname
5) list by age
0) quit
your choice: Thank you for using TwoPersonDataBase
